sys/vane/dill.hoon

```diff
@@ -459,9 +451,18 @@
                 ?=(^ j.hef.all)
             ==
           +>.$
-        %+  done(hef.all [~ ~ ~ ~ ~ ~ ~])
+        ::
+        =/  ven  hef.all
+        =>  .(hef.all [~ ~ ~ ~ ~ ~ ~])
+        ::
+        %+  done
           %mass
-        =>  [hef.all d=wegh]
+        =/  d=mass  :+  %dill  %|
+                    :~  hey+&+hey.all
+                        dug+&+dug.all
+                        dot+&+all
+                    ==
+        =,  ven
         [%vanes %| ~[u.a u.b u.c d u.e u.g u.f u.j]]
```

Comment:
I don't really like the `=,` here. Maybe `~[u.a u.b ...]:[ven .]`, although that might be confusing in it's own right. Alternately, you could `=.  u.d` instead of `=/  d` above, then `~[u.a u.b ...]:ven` would just work.

Reply:
I replaced the offending `=,` with a mutation into the result data structure.